src/onnx-mlir.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------ onnx-mlir.cpp - Compiler Driver  ------------------===//
+//
+// Copyright 2019-2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Main entry function for onnx-mlir driver.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Compiler/CompilerUtils.hpp"
+
+using namespace std;
+using namespace onnx_mlir;
+
+extern llvm::cl::OptionCategory OnnxMlirOptions;
+
+int main(int argc, char *argv[]) {
+  mlir::MLIRContext context;
+  registerDialects(context);
+
+  llvm::cl::opt<string> inputFilename(llvm::cl::Positional,
+      llvm::cl::desc("<input file>"), llvm::cl::init("-"),
+      llvm::cl::cat(OnnxMlirOptions));
+
+  llvm::cl::opt<string> outputBaseName("o",
+      llvm::cl::desc("Base path for output files, extensions will be added."),
+      llvm::cl::value_desc("path"), llvm::cl::cat(OnnxMlirOptions),
+      llvm::cl::ValueRequired);
+  llvm::cl::opt<EmissionTargetType> emissionTarget(
```

Comment:
Nit: new line between the options

Reply:
I assume this file is showing up as new, but is just main.cpp. It would still be nice to add the new line.

Reply:
Yes will do that. Is not a new file, I renamed it from `main.cpp` to `onnx-mlir.cpp` because I think the name is more descriptive. Also renamed the file we used as the main file in for `onnx-mlir-opt` to `onnx-mlir-opt.cpp` because is more descriptive and is more conforming to the naming convention used by the similar MLIR tool `mlir-opt`.  Hope you guys do not mind the renaming.

Reply:
done